egs/fisher_swbd/s5/local/chain/run_tdnn_lstm_1a_svd.sh

```diff
@@ -0,0 +1,376 @@
+#!/bin/bash
+# Saikiran Valluri, 2019, Govivace.Inc
+
+# The script is organized as below.
+# First we train the baseline LSTMP-TDNN config chain model for few epochs on the multien data,
+# Then, we perform SVD based refactoring of all the Affine components in this baseline final.mdl,
+# in order to reduce the overall model parameters size,
+# as determined by the bottleneck dim value or Energy and Shrinkage threshold values.
+# Then, we finetune the weight parameters of the refactored model using entire multien data for single epoch.
+
+# Command used for comparing  WER decoding on testsets :
+#  ./local/chain/compare_wer_general.sh --looped tdnn_lstm_1a_sp tdnn_lstm_1a_svd_sp
+
+
+# System                tdnn_lstm_1a_sp
```

Comment:
The WER stats in the script are for the NonSVD model - tdnn_lstm_1a_sp, and in order to facilitate comparison I included WER figures for some prominent experiments with the SVD retrained model, for example, like:
# WER on rt03(tg)
#  [SVD retrained]           12

Reply:
Did you run the command `./local/chain/compare_wer_general.sh --looped tdnn_lstm_1a_sp tdnn_lstm_1a_svd_sp` to get this? Because that's what the comment above says. If not, mention how you got these..

Reply:
Yes, we could derive the WER numbers by runnning the commented cmd line after running the entire script e2e.